numba_dpex/spirv_generator.py

```diff
@@ -19,11 +21,19 @@ def _raise_bad_env_path(msg, path, extra=None):
     raise ValueError(error_message)
 
 
-_real_check_call = check_call
-
-
-def check_call(*args, **kwargs):
-    return _real_check_call(*args, **kwargs)
+def run_cmd(args, error_message=None):
+    try:
+        check_output(
+            args,
+            stderr=STDOUT,
+        )
+    except CalledProcessError as err:
+        if error_message is None:
+            error_message = f"Error during call to {args[0]}"
+        raise InternalError(
```

Comment:
Nice! However, can we add an analogue of `numba.core.errors.InternalError` to numba_dpex.core.exceptions` and raise that exception?

The reason is it will help users understand that error came from within `numba-dpex` and not `numba`.